docs/introduction/how-to-work-with-money.md

```diff
@@ -0,0 +1,388 @@
+use Symfony\Component\Form\Extension\Core\Type\MoneyType;
+use Symfony\Component\Validator\Constraints\NotBlank;
+
+// ...
+
+$priceTableFormBuilder->add($key, MoneyType::class, [
+    'scale' => 6,
+    'required' => true,
+    'invalid_message' => 'Please enter price in correct format (a number with decimal separator)',
+    'constraints' => [
+        new NotBlank(['message' => 'Please enter price']),
+        new NotNegativeMoneyAmount(['message' => 'Price must be greater or equal to zero']),
+    ],
+]);
+```
+
+#### MoneyRange
+
+Similarly to [the Symfony `Range` constraint](https://symfony.com/doc/3.4/reference/constraints/Range.html), it validates that the amount of money is between some minimum and maximum.
+
+It has four options:
+- `min` specifies the minimum value, has to be an instance of `Money` or `null`
+- `max` specifies the maximum value, has to be an instance of `Money` or `null`
+- `minMessage` specifies the validation error message in case the entered value is less than the `min` value
+- `maxMessage` specifies the validation error message in case the entered value is greater than the `max` value
+
+At least one of the `min` and `max` options has to be provided for the constraint to make sense.
+
+```php
+use Shopsys\FormTypesBundle\MultidomainType;
+use Shopsys\FrameworkBundle\Component\Money\Money;
+use Shopsys\FrameworkBundle\Form\Constraints\MoneyRange;
+use Symfony\Component\Form\Extension\Core\Type\MoneyType;
+
+// ...
+
+$zboziFeedProductFormBuilder->add('cpc', MultidomainType::class, [
+    'label' => $this->translator->trans('Maximum price per click'),
+    'entry_type' => MoneyType::class,
+    'required' => false,
+    'entry_options' => [
+        'currency' => 'CZK',
+        'constraints' => [
+            new MoneyRange([
+                'min' => Money::create(1),
+                'max' => Money::create(500),
+            ]),
+        ],
+    ],
+]);
+```
+
+## Money in Twig Templates
+
+Instances of the `Money` class cannot be directly converted to strings.
+To be able to display a monetary value in a template you should use one of the prepared Twig filters.
+
+These filters can be used only with an instance of `Money`.
+
+### moneyFormat
+
+Formats the amount of money as a decimal number without any currency symbol.
+Optional parameters can be provided to specify the *number of decimal places* (`null` by default, meaning all), the *decimal point character* (`"."` by default) and the *separator of thousands* (`""` by default).
+
+```twig
+{# the "money" variable contains Money::create('1234.5670') #}
+
+{{ money|moneyFormat }}                  {# renders "1234.5670" #}
+{{ money|moneyFormat(0) }}               {# renders "1235" #}
+{{ money|moneyFormat(2, ',') }}          {# renders "1234,57" #}
+{{ money|moneyFormat(null, '.', ',') }}  {# renders "1,234.5670" #}
+```
+
+### price
+
+Filter `price` formats the amount of money in a localized manner.
```

Comment:
please add examples into all filters with rendered outputs. As same as you do with `moneyFormat`
It is really helpful for reader to imagine what is going on.

Reply:
I consider this out of scope of this PR. I've already incuded an example in the text:

> For example, one thousand Czech crowns would be rendered as *"CZK1,000.00"* on an English domain and as *"1 000,00 Kč"* on a Czech one.

There is a lot of price filters and the output depends on global configuration, current domain and whether you are in administration or not.

Reply:
yep, you mentioned it... but without examples i don't know how to use filters and what will the the output

Reply:
Due to the dependency on configuration, domain and whatnot I'd rather keep it as-is.,